bytesize talk dsl2 coding styles pt1 (#1168)
nf-core/nf-co.re: markdown/events/2022/bytesize-dsl2-coding-styles-pt1.md

---

> title: 'Bytesize: DSL2 Coding style recommendations pt1'

Reviewer: ```suggestion
title: 'Bytesize: DSL2 Coding style recommendations (Part 1)'
```

subtitle: Maxime Garcia - Barncancerfonden, Stockholm, Sweden
type: talk
start_date: '2022-07-05'
start_time: '13:00 CET'
end_date: '2022-07-05'
end_time: '13:30 CET'
location_url:
  - https://kth-se.zoom.us/j/68390542812
---

# nf-core/bytesize

Join us for our **weekly series** of short talks: **“nf-core/bytesize”**.

Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
These will be recorded and made available at <https://nf-co.re>
It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!

> ## Bytesize: DSL2 Coding style recommendations pt1

Reviewer: ```suggestion
## Bytesize: DSL2 Coding style recommendations (Part 1)
```

> This week, Maxime Garcia ([@MaxUlysse](https://github.com/MaxUlysse)) will explain us how he thinks DSL2 Coding style should look like. Join in into the discussion :)

Reviewer: ```suggestion
This week, Maxime Garcia ([@MaxUlysse](https://github.com/MaxUlysse)) will explain us what he thinks the perfect DSL2 coding style should be.

Come and join the ensuing discussion, it's going to be lively 😉 
```
